run/run.go

```diff
@@ -76,6 +76,8 @@ func GetMain(ctx context.Context) *cobra.Command {
 		},
 	}
 
+	cmd.PersistentFlags().BoolVar(&printer.DisableOutputTruncate, "disable-output-truncate", false,
```

Comment:
I think it would be better if this flag was called `truncate-output` and the default value was `true`.

Reply:
done